src/protocols/secure_channel/SessionIDAllocator.h

```diff
@@ -46,7 +49,7 @@ class SessionIDAllocator
     static constexpr uint16_t kMaxSessionID       = UINT16_MAX;
     static constexpr uint16_t kUnsecuredSessionId = 0;
 
-    uint16_t mNextAvailable = 1;
+    static uint16_t sNextAvailable;
```

Comment:
This is breaking Test-CHIP-SessionIDAllocator.  It appears initializing sNextAvailable to 1 would fix it.

Reply:
It is being initialized to 1, but it's failing because each test assumed the allocator's internal counter was reset to 1 upon construction. Fixing it now...